src/components/core/Tabs/Tabs.recipe.ts

```diff
@@ -62,6 +62,7 @@ export const tabsRecipe = defineSlotRecipe({
         },
       },
       _disabled: {
+        color: "fg.disabled",
         cursor: "not-allowed",
         _hover: {
           color: "fg.subtle",
```

Comment:
change this to `fg.disabled` as well. Also would like your thoughts on slightly adjusting the `semanticTokens` as this is difficult to read for sure. For a bit better readability, I propose changing it just a bit to:

```ts
fg: {
  ...
  disabled: {
    value: {
      base: "{colors.neutral.300}",
      _dark: "{colors.neutral.600}",
    },
  },
}

Reply:
Yeah, glad you brought it up. It would be ideal to change semantics instead of throw in one-offs.

Reply:
Hell yeah! Well if you want to make that adjustment in this PR since the issue is related / found in the `Tabs` that would be great! Just add in a second `changeset` describing the change to the `semanticTokens`.